pytorch_lightning/utilities/cli.py

```diff
@@ -213,6 +213,7 @@ def __init__(
         parser_kwargs: Dict[str, Any] = None,
         subclass_mode_model: bool = False,
         subclass_mode_data: bool = False,
+        run: bool = True,
```

Comment:
super nit: for other CLI tools i've seen `dry_run` be the term used. we could use it here too

Reply:
AFAIK, `dry_run` is sort of a "test it tworks" flag before doing the actual run

However, this `run` flag isn't for those who want to test their parsing, but for those who don't want to automatically start fitting their model.

Also notice that this flag is not exposed via command line. So you couldn't do

```python
# see if it parses
python script.py --dry_run True

# actual fit
python script.py
```

You'd need to change the file's flag.